scripts/configure_dev-deps.sh

```diff
@@ -31,6 +31,11 @@ function install_go_module {
     fi
 }
 
+UNAME=$(uname)
+if [[ "${UNAME}" == *"MINGW"* ]]; then
+	export GOPATH=$HOME/go
+fi
+
```

Comment:
I don't feel that this is the right thing to do. It seems to be windows specific, and it's would be better if we could align all the platforms. Given that this file being invoked via configure_dev.sh, which being invoked manually, I would suggest adding this to the caller. Also, if we want to avoid taking this to the caller, we could add a make larget for this one, since the makefile already export the GOPATH correctly.

Reply:
I have the doubt if this script is always meant to be executed by the parent script or if it can be executed standalone. Please confirm me it is always the former, so I'll do the change

Reply:
At this time, the script is always meant to be invoked by `script/configure_dev.sh`; there is no use case that I can see where the script need to be invoked on its own. ( i.e. if any of the dependencies changes, the user would run the `script/configure_dev.sh` which would update all the required dependencies ).

The `script/configure_dev.sh` doesn't really care about go, so we could definitely add a check there to ensure that the GOPATH was previously set by the user.
I think that it would get solved better by adding a "configure" target to make.
i.e.
configure:
    ./scripts/configure_dev.sh

Reply:
Ok let me see it on the weekend and see necessary changes to documentation.

Reply:
thanks! I really appreciate your contribution !

Reply:
You are welcome 👍